src/RelayEnvironment.ts

```diff
@@ -0,0 +1,17 @@
+import { Environment, Network, RecordSource, Store } from "relay-runtime";
+import fetchGraphQL from "./fetchGraphQL";
+
+// Relay passes a "params" object with the query name and text. So we define a helper function
+// to call our fetchGraphQL utility with params.text.
+async function fetchRelay(params: any, variables: any) {
+  console.log(
+    `fetching query ${params.name} with ${JSON.stringify(variables)}`
+  );
+  return fetchGraphQL(params.text, variables);
+}
+
+// Export a singleton instance of Relay Environment configured with our network function:
+export default new Environment({
```

Comment:
followed the [guide here](https://relay.dev/docs/getting-started/step-by-step-guide/)